crux-core/src/crux/api/alpha/NodeStatus.java

```diff
@@ -0,0 +1,47 @@
+package crux.api.alpha;
+
+import clojure.lang.Keyword;
+
+import java.util.Map;
+
+public class NodeStatus {
+    public final String version;
```

Comment:
we should consider which of these we want to support, and whether we want to support them as a separate object or 'just' values on the node itself, in Java.

maybe not 100% either way, too - version, revision, indexversion etc are all static when the node's created; the others are dynamic

Reply:
Yeah - this was a sort of placeholder, one of the classes I wanted a bit more thinking on (same as EntityTx)